pkg/job/v1alpha3/job/job.go

```diff
@@ -28,9 +28,10 @@ func (j Job) GetDeployedJobStatus(
 			log.Error(unerr, "NestedFieldCopy unstructured to status error")
 			return nil, unerr
 		}
-		log.Info("NestedFieldCopy unstructured to status error",
-			"err", "Status is not found in job")
-		return &jobCondition, nil
+		// Job does not have the running condition in status, thus we think
+		// the job is running when it is created.
```

Comment:
Can you explain this comment? This is little confusing. When there is no status field, it means that job is just created.

Reply:
Yeah, The Job does not have a running condition, thus we cannot use the same logic as TFJob